src/sqlfluff/rules/L028.py

```diff
@@ -32,20 +37,24 @@ class Rule_L028(Rule_L020):
     .. code-block:: sql
 
         SELECT
-            a,
-            b
+            foo.a,
+            foo.b
         FROM foo
 
         -- Also good
 
         SELECT
-            foo.a,
-            foo.b
+            a,
+            b
```

Comment:
Any reason you changed this? Or just reflecting your preference?

Reply:
I was having linting issues with black (which actually turned out not to be this line) my bad :)

Reply:
Thought you were going to revert this?